buildout.cfg

```diff
@@ -12,6 +12,11 @@ parts +=
     i18ndude
     omelette
     rebuild_i18n-sh
+    webpack
```

Comment:
it's time to create a recipe; we don't want to have to add all of this boilerplate in all packages.